Dockerfile

```diff
@@ -12,15 +13,19 @@ RUN cargo chef prepare --recipe-path recipe.json
 
 FROM base as build
 
+# require a running postgres instance on the host
+ENV DATABASE_URL=postgres://bors:bors@host.docker.internal/bors
+
 WORKDIR /app
 COPY --from=planner /app/recipe.json recipe.json
 
 RUN cargo chef cook --release --recipe-path recipe.json
+RUN cargo sqlx migrate run
```

Comment:
At this point the `migrations` directory hasn't been copied to the image yet. I with we could just disable the compile-time check in the Dockerfile (is it possible?). It is annoying that a DB is needed for the Docker image build. We upload the built image to Amazon ECS, so we would also need to make Postgres available [here](https://github.com/rust-lang/bors/blob/main/.github/workflows/deploy.yml#L32).

Do you know if there is a way to completely disable the compile-time checks?

Reply:
We will need to run the migrations when the image is executed (so that they run against the production DB), not during the build of the container itself.

Reply:
Yeah, it should probably be an argument.

Reply:
I don't know how to connect to localhost during docker build 🥹

Reply:
I guess that there are ways around that (https://www.reddit.com/r/rust/comments/18y4ts3/compile_time_issue_with_rustpostgres_docker/ maybe?), but I wonder if passing the `.sqlx` to it might not be an easier solution after all. We don't need to commit it, we could just prepare it on CI and then pass it to the Dockerfile.

What issues with `cargo sqlx prepare` did you encounter?

Reply:
Now that I think about it, most of the time, it was due to the the combination of mismatched versions between our dev machines and CI, along with cargo `-D warnings` flag, making each `cargo sqlx prepare` as miserable as it could be.